src/main/java/lab3/game/Board.java

```diff
@@ -0,0 +1,68 @@
+package lab3.game;
+
+public class Board {
+    // TODO: encapsulate the representation of the tictactoe board and provide instance methods to access and update it
+    private char[][] board;
+    private final int SIZE = 3;
+
+    public Board() {
+        board = new char[SIZE][SIZE];
+        initializeBoard();
+    }
+
+    private void initializeBoard() {
+        for (int i = 0; i < SIZE; i++) {
+            for (int j = 0; j < SIZE; j++) {
+                board[i][j] = ' ';
+            }
+        }
+    }
+
+    public boolean placeMark(int row, int col, char player) {
```

Comment:
This works, but if you had used a specific type that only allows valid row/col values, then you wouldn't need the checks comparing `SIZE` at all.  See the sample solution.